service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisTemplateImages.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class DocmosisTemplateImages {
+    private final String draftWaterMark;
+    private final String courtseal;
```

Comment:
Court seal is really two words, I can understand not wanting to change in the templates but we can use the json property here to specify the name whilst maintaining camel case in the java model